examples/getting_started/4-Gridded_Datasets.ipynb

```diff
@@ -85,38 +86,48 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Other datatypes"
+    "Instead of defining the coordinates manually, we recommend using [xarray](http://xarray.pydata.org/en/stable/), which will flexibly work with labeled n-dimensional arrays. We can even make a clone of our dataset and set the datatype to xarray to convert to an ``xarray.Dataset``, which is the recommended format for gridded data in HoloViews. In this instance, you can view the converted xarray (if `xarray` is installed) using:\n",
+    "\n",
+    "```python\n",
+    "ds.clone(datatype=['xarray']).data\n",
+    "```\n"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Instead of defining the coordinates manually, we recommend using [xarray](http://xarray.pydata.org/en/stable/), which will flexibly work with labeled n-dimensional arrays. We can even make a clone of our dataset and set the datatype to xarray to convert to an ``xarray.Dataset``, which is the recommended format for gridded data in HoloViews:"
+    "To see more details on working with different datatypes have a look at the [user guide](../user_guide/08-Gridded_Datasets.ipynb)."
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
+   "cell_type": "markdown",
    "metadata": {},
-   "outputs": [],
    "source": [
-    "ds.clone(datatype=['xarray']).data"
+    "### Viewing the data\n",
+    "\n",
+    "The first thing we will do is customize the appearance of the elements used in the rest of the notebook. For more information about `opts.defaults`, please consult the [User Guide](../user_guide/03-Customizing_Plots.ipynb) for more information."
    ]
   },
   {
-   "cell_type": "markdown",
+   "cell_type": "code",
+   "execution_count": null,
    "metadata": {},
+   "outputs": [],
    "source": [
-    "To see more details on working with different datatypes have a look at the [user guide](../user_guide/08-Gridded_Datasets.ipynb)."
+    "opts.defaults(\n",
+    "    opts.Spread(width=600),\n",
+    "    opts.GridSpace(shared_xaxis=True, shared_yaxis=True),\n",
+    "    opts.Image(cmap='viridis', width=400, height=400),\n",
+    "    opts.Path(color='white'),\n",
+    "    opts.Labels(text_color='white', text_font_size='8pt', text_align='left', text_baseline='bottom'),\n",
+    "    opts.Overlay(show_legend=False))"
```

Comment:
Should we agree to list things in alphabetical order? If so it would be good to change this. Also while I agree defaults are often useful I worry that putting a huge block of options like this is intimidating and hard to understand since it's unclear what it will apply to.

Reply:
The current recommendation is alphabetical order with containers at the end (`Layout`, then `Overlay`). So in the above I need to swap the positions of `Path` and `Labels`.

Reply:
Actually, there are more elements mentioned here I need to sort.

Reply:
The final ordering is:

```python
opts.defaults(
    opts.GridSpace(shared_xaxis=True, shared_yaxis=True),
    opts.Image(cmap='viridis', width=400, height=400),
    opts.Labels(text_color='white', text_font_size='8pt', text_align='left', text_baseline='bottom'),
    opts.Path(color='white'),
    opts.Spread(width=600),
    opts.Overlay(show_legend=False))
```

Reply:
Fixed in 3126858